src/src/DataStructs/NodeStruct.h

```diff
@@ -17,6 +17,11 @@
 
 const __FlashStringHelper * getNodeTypeDisplayString(uint8_t nodeType);
 
+#ifndef FEATURE_ESP_P2P
+  #error "MFD"
+#endif
```

Comment:
Was that used for debugging, or is the intent to change it into a little more descriptive message? 😉

Reply:
I think you're looking at an older commit, as it doesn't seem to be there in the latest commit?

Reply:
Ah, yes, looked at separate commits, so NP.